File: Frontend/src/models/GetRoomResponse.ts
import {GetRoomEquipmentResponse} from "@/models/GetRoomEquipmentResponse";

export interface GetRoomResponse {
    id: string;
    name: string;
    capacity: number;
    floor: number;
    description: string;
    isActive: boolean;
    photo: string;
    roomEquipments: GetRoomEquipmentResponse[];
}

File: Backend/MeetSpace/MeetSpace.Api/Contracts/Rooms/GetRoomResponse.cs
using MeetSpace.Api.Contracts.RoomEquipment;

namespace MeetSpace.Api.Contracts.Rooms;

public class GetRoomResponse
{
    public Guid Id { get; set; }
    public string Name { get; set; } = null!;
    public uint Capacity { get; set; }
    public int Floor { get; set; }
    public string? Description { get; set; }
    public string Photo { get; set; } = null!;
    public List<GetRoomEquipmentResponse> RoomEquipments { get; set; } = [];
}